.appveyor.yml

```diff
@@ -60,45 +59,11 @@ environment:
 matrix:
     fast_finish: true
 install:
-    - systeminfo
-    - c:\cygwin\bin\uname -a
-    - c:\cygwin\bin\cat /proc/cpuinfo
-    - c:\cygwin\bin\cat /proc/meminfo
-    - c:\cygwin\bin\ls -l /cygdrive/c
-    - c:\cygwin\bin\ls -l "/cygdrive/c/Program Files (x86)"
-    - c:\cygwin\bin\ls -l "/cygdrive/c/Program Files"
-    - c:\cygwin\bin\ls -l /cygdrive/c/Tools
-    - c:\cygwin\bin\ls -l /cygdrive/c/Libraries
-    - c:\cygwin\bin\ls -l /cygdrive/c/ProgramData
-    - c:\cygwin\bin\env
-    - nuget install Gettext.Tools -OutputDirectory c:\deps
-    - c:\cygwin\bin\find /cygdrive/c/deps "-type" f
     - ps: |
-        if ((Test-Path "c:\OpenSSL-$env:ssl_arch") -And ((Get-Item "c:\OpenSSL-$env:ssl_arch\bin\ssleay32.dll").VersionInfo.ProductVersion -eq '1.0.2o')) {
```

Comment:
please reinstate the openssl handling entirely. AVs pre-installed one is outdated and the whole check is there so it does nothing if at any point in time the version on AV is the same one as the installed one.